internal/collector/pgadmin.go

```diff
@@ -102,7 +101,7 @@ func EnablePgAdminLogging(ctx context.Context, spec *v1beta1.InstrumentationSpec
 		Processors: []ComponentID{
 			"resource/pgadmin",
 			"transform/pgadmin_log",
-			SubSecondBatchProcessor,
+			LogsBatchProcessor,
 			CompactingProcessor,
 		},
 		Exporters: exporters,
```

Comment:
❓ This and the pipeline below it differ only in their receivers. Is that correct?

Reply:
Yeah, iirc, the only distinction in the `transform` step is a pattern match that will hit for pgadmin and not for gunicorn (or vice versa) -- so, yeah, we could combine them.